i18next-parser.config.ts

```diff
@@ -0,0 +1,19 @@
+export default {
+  output: 'src/locales/$LOCALE/$NAMESPACE.json',
+  locales: [ 'en', 'ja', 'he' ],
+
+  // Because some keys are dynamically generated, i18next-parser can't detect them.
+  // We add these keys manually, so we don't want to remove them.
+  keepRemoved: true,
+
+  // We use plain English keys, so we disable key and namespace separators.
+  keySeparator: false,
+  namespaceSeparator: false,
+
+  defaultValue: (locale, ns, key) => {
+    if (locale === 'en') {
+      return key;
+    }
+    return '__STRING_NOT_TRANSLATED__';
```

Comment:
I would keep the English translation as default.

Reply:
I was thinking about returning the key for every language in case the string is not found, and maybe calling a console.warn or something.
Your call, I'm good with either solution you decide.

Reply:
I'll do a console.warn